index.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var _ = require('lodash');
+var path = require('path');
+var fs = require('fs');
+var through2 = require('through2');
+var gettextParser = require('gettext-parser');
+var Handlebars = require('handlebars');
+
+
+var replaceText = function(catalog, opts, chunk, enc, callback) {
+  var template = _.template(chunk.toString(), {
```

Comment:
If we run into a situation where we need to increase the call stack, then lets do something. I Googled around for lodash templates callstack exceeded and didn't find much. Hopefully the issue you were running into was specific to the context in which lodash was being used.